app/forms/openid_connect_authorize_form.rb

```diff
@@ -116,7 +116,7 @@ def aal
   end
 
   def aal_value
-    aal_values.sort.max
+    highest_level_aal(aal_values)
```

Comment:
[suggestion] this code works, which is great, but i think i would suggest reordering it a little bit!

i would suggest making the aals_by_priority their own method, and then doing the "find" in this method. ie:

```ruby
def aal_value
  aals_by_priority.find { |aal|  aal_values.include?(aal) }
end

def aals_by_priority
  [
    Saml::Idp::Constants::AAL2_HSPD12_AUTHN_CONTEXT_CLASSREF,
    Saml::Idp::Constants::AAL3_HSPD12_AUTHN_CONTEXT_CLASSREF,
    Saml::Idp::Constants::AAL2_PHISHING_RESISTANT_AUTHN_CONTEXT_CLASSREF,
    Saml::Idp::Constants::AAL3_AUTHN_CONTEXT_CLASSREF,
    Saml::Idp::Constants::AAL2_AUTHN_CONTEXT_CLASSREF
  ]
end
```

Reply:
Went with making aals_by_priority a constant. frozen at that.

Reply:
👏 sweet!